### PR TITLE
Help update

### PR DESCRIPTION
**Pull request by outofnamespace**
Edit
- All files with GCS version stamp "Last updated for v5.7.0"
- Renamed all lower case filenames and updated all related links
- Markdown.md and Note.md link to markdown guide
- Library Tree and common icon are no longer deadends
- Add help Help to Overview and common icon
- Try using a dictionary this time

Added
- calc.png
- added q.png

Replaced
- pen-dark.png

### CODE REVIEW

File: Library/Markdown/Help/Interface/Character Sheet.md [hunk 1]
```diff
@@ -55,7 +57,7 @@ Select the character sheet to modify:
   - **\<context_menu> -> Delete**
   - Press 'Del'. It might say 'delete', but that's a pretty long word for such a little key.
 
-When adding elements, some present dialogues to get user input for [mods](./mods.md "Mods") (e.g: Riding skill which requires specialization, Language trait requires a specialization and modifiers). Substitutions are presented as a field that needs completion. Modifications are a checklist of variables. Set the values and select Ok. Select Cancel to dismiss the dialogue, but that likely leaves the element and character incomplete. Edit the element to complete the selections.
+When adding elements, some present dialogues to get user input for [mods](./Mods.md "Mods") (e.g: Riding skill which requires specialization, Language trait requires a specialization and modifiers). Substitutions are presented as a field that needs completion. Modifications are a checklist of variables. Set the values and select Ok. Select Cancel to dismiss the dialogue, but that likely leaves the element and character incomplete. Edit the element to complete the selections.
```
Comment (richardwilkes) [inline Library/Markdown/Help/Interface/Character Sheet.md:60]: “dialogues” should be “dialogs”

File: Library/Markdown/Help/Interface/Character Sheet.md [hunk 2]
```diff
@@ -97,7 +99,9 @@ Otherwise, the file needs to be deleted with the OS file manager.
 The calculators are for Jumping, Throwing and Hiking. The results are based in character sheet settings and calculator options.
 
 ## Fields
-Some fields are [free-form data](./free-form%20data.md "free-form data") and some are formatted. Some of the formats depend on the sheet settings. Some fields are calculated and can't be directly changed. Some fields are lists. To *some* it up, there are a lot of fields.
+*Almost all character sheet data is [unlinked](./Unlinked%20Data.md "Unlinked Data").*
+
+Some fields are [free-form data](./Free-Form%20Data.md "Free-Form Data") and some are formatted. Some of the formats depend on the sheet settings. Some fields are calculated and can't be directly changed. Some fields are lists. To *some* it up, there are a lot of fields.
```
Comment (richardwilkes) [inline Library/Markdown/Help/Interface/Character Sheet.md:104]: “*some*” should be “*sum*”

File: Library/Markdown/Help/Interface/Points Record.md [hunk 3]
```diff
@@ -6,11 +8,11 @@ The Points Record is a list of character point changes. The first record is the
 Alter the current record(s) or add a new one.
  It's all done with the tool buttons.
 
-## Interface
-### Tool Bar
-- **[Save](./common%20icon.md "common icon:Save") ( ![](./img/check.png "Save") )** : save and close
-- **[Cancel](./common%20icon.md "common icon:Cancel") ( ![](./img/no.png "Cancel") )** : cancel and close
-- **[Add](./common%20icon.md "common icon:Add") ( ![](./img/plus.png "Add") )** : add a record
+## InterfaceCommon Icon:
```
Comment (richardwilkes) [inline Library/Markdown/Help/Interface/Points Record.md:11]: This looks like a typo, as does the next line.

File: Library/Markdown/Help/Interface/Settings.md [hunk 4]
```diff
@@ -9,7 +11,7 @@ Settings can be exported and imported to sets, which will appear under the setti
 **\<library>\\<settings.file>**\
 **\<library>\\Settings\\<settings.file>**
 
-When using the Settings sub-diretory, further sub-directories are possible, to any depth. Though you should stop and ask yourself, "why?", before doing that. Menu list naming is based on the filename, and the setting type is based on the file extension. Duplicate names are shown. Regardless of the directory structure, all settings are displayed in a single menu column, without indication of file location.
+When using the Settings sub-directory, further sub-directories are possible, to any depth. Though you should stop and ask yourself, "why?", before doing that. Menu list naming is based on the filename, and the setting type is based on the file extension. Duplicate names are shown. Regardless of the directory structure, all settings are displayed in a single menu column, without indication of file location.
```
Comment (richardwilkes) [inline Library/Markdown/Help/Interface/Settings.md:14]: Not completely true. The library the file is in is identified.

File: Library/Markdown/Help/Interface/Sheet Settings.md [hunk 5]
```diff
@@ -29,7 +31,10 @@ The menu bar option is available when a character sheet is active. Changes take
 - **Page Settings** : Used in printing and exporting
 - **Block Layout** : Arrangement of named blocks on the character sheet
   - block names are case-insensitive but otherwise immutable
-  - blocks can be removed by not including the name (e.g.: spells, for a non-magic campaign)
+  - blocks can two side by side or in a single column
```
Comment (richardwilkes) [inline Library/Markdown/Help/Interface/Sheet Settings.md:34]: “blocks can two side” -> “blocks can be arranged two side”

File: Library/Markdown/Help/Interface/Trait.md [hunk 6]
```diff
@@ -19,7 +21,7 @@ Traits use the [Detail Editor](./Detail%20Editor.md "Detail Editor").
 
 Control Roll does not affect calculations, but it does display under the trait on the trait list.
 
-The Container Type can impact how character points are distributed. *Group* means it's just a trait container for holding stuff. Like your lunch. *Meta-Trait* containers allow all the container traits to stay together, as listed (B262). Buying off just one is probbaly not allowed. Making it a *Race* container, causes the Points field to count the contents of the container against the Race value and activates the Ancestry field. *Alternative Abilities* tends toward being a Supers thing (B61 + Supers).
+The Container Type can impact how character points are distributed. *Group* means it's just a trait container for holding stuff. Like your lunch. *Meta-Trait* containers allow all the container traits to stay together, as listed (B262). Buying off just one is probably not allowed. Making it a *Race* container, causes the Points field to count the contents of the container against the Race value and activates the Ancestry field. *Alternative Abilities* tends toward being a Supers thing (B61 + Supers).
```
Comment (richardwilkes) [inline Library/Markdown/Help/Interface/Trait.md:24]: There is a new container type in v5.7: “Attribute” (hmm… might be plural — I’m not someplace I can easily check right now). Anyway, this works just like the Race container does, assigning its point total towards the Attributes portion of the point breakdown.

**Comment by outofnamespace** [comment]
1) that's what I get for using a dictionary
2) way to ruin a joke
3) my keyboard must have been on backwards
4) I knew that and was supposed to fix it and obviously that worked out well
5) yea   weird wording
6) I saw something about an attributes container but didn't add it as I knew nothing about it

**Review by richardwilkes — APPROVED** [review]

**Pull request merged**